builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -242,6 +247,11 @@ node() {
 					echo "setting lambda execution role as ${configLoader.AWS.ROLEID}";
 					setLambdaExecutionRole(configLoader.AWS.ROLEID);
 
+          def isLambdaUpdateRequired
```

Comment:
indentation here and other places in the code

Reply:
showing correct in my ide

Reply:
done